README.md

```diff
@@ -18,8 +18,7 @@ repositories {
 3. Add the dependency under ```dependencies```
 ```
 implementation 'com.rudderstack.android.sdk:core:[1.0,)'
-implementation 'com.rudderstack.android.integration:braze:1.0.4'
-implementation 'com.appboy:android-sdk-ui:10.0.0'
+implementation 'com.rudderstack.android.integration:braze:1.0.5'
```

Comment:
We should increase to 2.0.0 as this is a breaking change.!

Reply:
Implementing that. Thanks.